src/advanced_examples.md

```diff
@@ -494,15 +449,17 @@ type box<V> {
 }
 
 on start fn {
-  let int8Box: box<int8>
-  int8Box.val = 8
-  int8Box.set = true
+  let int8Box = new box<int64> {
```

Comment:
why was the reasoning for prefering `box` over `Box`?

Reply:
Oh, just because capitalizing the generic types is just a convention, and I wanted to show that you don't *have* to if you don't want to. But I do agree it's ugly. It's also a hidden away place and not explicitly called out, so I'm not sure if that matters that much.

Reply:
Then let's point out the fact that the convention is not enforced, but probably just go with `Box`